styles.css

```diff
@@ -0,0 +1,72 @@
+*{
+    margin: 0;
+    
+}
+.app {
+    padding: 50px;
+    background: #1F2937;
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+    align-items: center;
+   
+}
+.header {
+    display: flex;
+    flex-direction: column;
+    gap: 10px; 
+    
+}
+h1{
+    font-family: "Karla",sans-serif;
+    font-weight: 800;
+    font-size: 40px;
+    line-height: 40px;
+    letter-spacing: -0.025em;
+}
+.top {
+    color: #FFFFFF;
+}
+.bottom {
+    color: #4ADF86;
+}
+p {
+    color: #D5D4D8;
+    font-family: "Inter",sans-serif;
+}
```

Comment:
here same as i said above, better to use class name for styling instead of using html elements. If all of your p have same properties then you can use same class name in your implementation